lectures/L11-slides.tex

```diff
@@ -94,11 +94,13 @@
 \begin{frame}
 \frametitle{Functional Dependency Theory Example}
 
-There are three:
+There are five:
 \begin{itemize}
-	\item $A \rightarrow H$ which is found by transitivity ($A \rightarrow B$ and $B \rightarrow A$).
+	\item $A \rightarrow H$ which is found by transitivity ($A \rightarrow B$ and $B \rightarrow H$).
+	\item $A \rightarrow BC$ which is found by the union rule ($A \rightarrow B$ and $A \rightarrow C$).
+	\item $A \rightarrow BCH$ which is found by the union rule ($A \rightarrow BC$ and $A \rightarrow H$).
 	\item $CG \rightarrow HI$ which is found by the union rule ($CG \rightarrow H$ and $CG \rightarrow I$).
```

Comment:
This line referenced an FD that wasn't present in the example in the previous slide.

Reply:
CG -> I